README.md

```diff
@@ -11,31 +52,50 @@ This repository includes files with plain SQL that can be used to recreate a dat
 - Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
 
 
-## Authors
+<!-- AUTHORS -->
+
+## 👥 Author <a name="author"></a>
+
 
-👤 **Author1**
+👤 **Recillah Khamala**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@Recillah-khamala](https://github.com/Recillah-Khamala)
+- Twitter: [@recillahk](https://twitter.com/recillahk)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/recillah-khamala/)
 
 
-## 🤝 Contributing
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Kindly consider adding a `Future Features` section and including 1-3 potential features to be added.

Reply:
You have given me change required to add features to a PostgreSQL project. I understand that this is a backend project and it is for me to understand how to build DB, tables, populate them and build relationships between the tables. What I don't understand is the need to add features because no one is going to interact with this part of the project.
Please help me understand what I need to add in the features section in this case.